modules/rosa-planning-considerations.adoc

```diff
@@ -1,22 +1,25 @@
 
 // Module included in the following assemblies:
 //
-// rosa_planning/rosa-planning-environment.adoc
+// rosa_planning/rosa-limits-scalability.adoc
 
-[id="initial-planning-considerations_{context}"]
-= Initial planning considerations
+[id="control-plane-and-infra-node-sizing-and-scaling_{context}"]
+= Control plane and infrastructure node sizing and scaling
 
-Consider the following tested object maximums when you plan your {product-title} cluster.
+When you install a {product-title} (ROSA) cluster, the number of required control plane and infrastructure nodes is automatically determined by the compute node count.
 
-These guidelines are based on a cluster of 102 workers in a multi-availability zone configuration. For smaller clusters, the maximums are lower.
+If you change the number of compute nodes in your cluster after installation, the Red Hat Site Reliability Engineering (SRE) team scales the control plane and infrastructure nodes as required, to maintain cluster stability.
 
-The sizing of the control plane and infrastructure nodes is dynamically calculated during the installation process, based on the number of worker nodes. If you change the number of worker nodes after the installation, control plane and infra nodes must be resized manually. Infra nodes are resized by the Red Hat SRE team, and you can link:https://access.redhat.com/[open a ticket in the Customer Portal] to request the infra node resizing.
+[id="node-sizing-during-installation_{context}"]
+== Node sizing during installation
 
-The following table lists the size of control plane and infrastructure nodes that are assigned during installation.
+During the installation process, the sizing of the control plane and infrastructure nodes is dynamically calculated. The sizing calculation is based on the number of compute nodes in a cluster.
```

Comment:
```suggestion
During the installation process, the sizing of the control plane and infrastructure nodes are dynamically calculated. The sizing calculation is based on the number of compute nodes in a cluster.
```

Reply:
Thank you for highlighting this. Resolved.